Microsoft.Toolkit.Win32/Microsoft.Toolkit.Sample.Forms.App/Form1.Designer.cs

```diff
@@ -0,0 +1,199 @@
+        private void InitializeComponent()
+        {
+            this.tabControl1 = new System.Windows.Forms.TabControl();
+            this.inkingTab = new System.Windows.Forms.TabPage();
+            this.inkToolbar1 = new Microsoft.Toolkit.Forms.UI.Controls.InkToolbar();
+            this.inkCanvas1 = new Microsoft.Toolkit.Forms.UI.Controls.InkCanvas();
+            this.webTab = new System.Windows.Forms.TabPage();
+            this.webViewCompatible1 = new Microsoft.Toolkit.Forms.UI.Controls.WebViewCompatible();
+            this.mediaTab = new System.Windows.Forms.TabPage();
+            this.mediaPlayerElement1 = new Microsoft.Toolkit.Forms.UI.Controls.MediaPlayerElement();
+            this.testTab = new System.Windows.Forms.TabPage();
+            this.inkToolbarCustomToolButton1 = new Microsoft.Toolkit.Forms.UI.Controls.InkToolbarCustomToolButton();
+            this.tabControl1.SuspendLayout();
+            this.inkingTab.SuspendLayout();
+            this.inkToolbar1.SuspendLayout();
+            this.webTab.SuspendLayout();
+            this.mediaTab.SuspendLayout();
+            this.SuspendLayout();
+            // 
+            // tabControl1
+            // 
+            this.tabControl1.Controls.Add(this.inkingTab);
+            this.tabControl1.Controls.Add(this.webTab);
+            this.tabControl1.Controls.Add(this.mediaTab);
+            this.tabControl1.Controls.Add(this.testTab);
+            this.tabControl1.Dock = System.Windows.Forms.DockStyle.Fill;
+            this.tabControl1.Location = new System.Drawing.Point(0, 0);
+            this.tabControl1.Name = "tabControl1";
+            this.tabControl1.SelectedIndex = 0;
+            this.tabControl1.Size = new System.Drawing.Size(1588, 1088);
+            this.tabControl1.TabIndex = 0;
+            // 
+            // inkingTab
+            // 
+            this.inkingTab.Controls.Add(this.inkToolbar1);
+            this.inkingTab.Controls.Add(this.inkCanvas1);
+            this.inkingTab.Location = new System.Drawing.Point(4, 29);
+            this.inkingTab.Name = "inkingTab";
+            this.inkingTab.Padding = new System.Windows.Forms.Padding(3);
+            this.inkingTab.Size = new System.Drawing.Size(1580, 1055);
+            this.inkingTab.TabIndex = 1;
+            this.inkingTab.Text = "Ink Canvas / Toolbar";
+            this.inkingTab.UseVisualStyleBackColor = true;
+            // 
+            // inkToolbar1
+            // 
+            this.inkToolbar1.Controls.Add(this.inkToolbarCustomToolButton1);
+            this.inkToolbar1.Dock = System.Windows.Forms.DockStyle.Top;
+            this.inkToolbar1.Location = new System.Drawing.Point(3, 3);
+            this.inkToolbar1.MinimumSize = new System.Drawing.Size(20, 60);
+            this.inkToolbar1.Name = "inkToolbar1";
+            this.inkToolbar1.Size = new System.Drawing.Size(1574, 60);
+            this.inkToolbar1.TabIndex = 2;
+            this.inkToolbar1.TargetInkCanvas = null;
```

Comment:
This will fail to change ink attributes like brush type. set to inkcanvas1
```suggestion
            this.inkToolbar1.TargetInkCanvas = this.inkCanvas1;
```

Reply:
done as I suggested